homeassistant/components/prosegur/strings.json

```diff
@@ -0,0 +1,22 @@
+{
+  "title": "Prosegur Alarm",
```

Comment:
As this is a brand, I think this doesn't need to be translated and should be removed from this file.